main.go

```diff
@@ -28,6 +30,13 @@ func main() {
 		glog.Exitf("Configuration could not be loaded or did not pass validation: %v", err)
 	}
 
+	// Create a soft memory limit on the total amount of memory that PBS uses to tune the behavior
+	// of the Go garbage collector. In summary, `cfg.GarbageCollectorLimit` serves as a fixed cost
+	// of memory that is going to be held garbage before a garbage collection cycle is triggered.
+	// This amount of virtual memory won’t translate into physical memory allocation unless we attempt
+	// to read or write to the slice below, which PBS will not do.
+	garbageCollectionLimit = make([]byte, cfg.GarbageCollectorLimit)
```

Comment:
We'll need to test this works as intended. We saw the following recommendation from a Medium article:
```
func main() {
	ballast := make([]byte, 2<<30)
	defer runtime.KeepAlive(ballast)
}
```

Reply:
Going with the `defer runtime.KeepAlive(garbageCollectionLimit)` recommendation then.